spec/models/card_spec.rb

```diff
@@ -3,11 +3,14 @@
 describe Card do
   it "original and translated texts cannot be equal" do
     expect { Card.create!(original_text: "test",
-                          translated_text: "test")
+                          translated_text: "test",
+                          user_id: 1)
     }.to raise_error
   end
 
-  let(:card) { Card.create!(original_text: "test", translated_text: "тест") }
+  let(:card) { FactoryGirl.create(:card, original_text: "test",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.